src/utils/data-management/loadRoles.js

```diff
@@ -0,0 +1,17 @@
+import fetchAPI from 'utils/http/fetchAPI';
```

Comment:
loadRoles and the other fetch files could be in their own folder `accessFetches` or something like that